src/views/InvolvementProfile/components/Membership/components/AdminCard/index.jsx

```diff
@@ -19,13 +19,11 @@ import { useParams } from 'react-router';
 import involvementService from 'services/activity';
 import membershipService from 'services/membership';
 import { stripDomain } from 'services/utils';
-import { gordonColors } from 'theme';
 import RequestsReceived from './components/RequestsReceived';
 import { ConflictError, NotFoundError } from 'services/error';
+import styles from './AdminCard.module.css';
 
 const headerStyle = {
-  backgroundColor: gordonColors.primary.blue,
-  color: '#FFF',
   padding: '10px',
```

Comment:
You should also add the other CSS tags into the external CSS, like padding.  There should be know style objects defined inside of code, they should be fully moved to separate CSS.